aio_pika/connection.py

```diff
@@ -195,17 +195,17 @@ async def __aexit__(
 
 
 def make_url(
-    url: Union[str, URL] = None,
+    url: Union[str, URL, None] = None,
     *,
     host: str = "localhost",
     port: int = 5672,
     login: str = "guest",
     password: str = "guest",
     virtualhost: str = "/",
     ssl: bool = False,
-    ssl_options: dict = None,
-    client_properties: FieldTable = None,
-    **kwargs: Any
+    ssl_options: Optional[Dict[str, Any]] = None,
```

Comment:
The better solution is using TypedDict like this:

```python
class SSLOptions(TypedDict):
    cafile: Optional[str]
    capath: Optional[str]
    cadata: Optional[str]
    keyfile: Optional[str]
    certfile: Optional[str]
    no_verify_ssl: Optional[int]


def make_url(
     ...
     ssl_options: Optional[SSLOptions] = None,
     ...
):
    ...
```

Reply:
The `SSLOptions` definition might be placed in abc.py

Reply:
Done, perhaps it's better to use `total=False` and remove Optional values in the SSLOptions class?